installation.md

```diff
@@ -1,53 +1,85 @@
 # Installation
 
-Deployer can be installed in two ways: using phar archive and using composer.
+Deployer can be installed in three ways: 
+1) using **composer package with phar only** - later called also "Deployer Distribution" version (no dependencies)
+2) using **composer package with sources** - later called also "Deployer Source" version (dependencies but good for debug 
+   Deployer) 
+3) using direct download of **phar archive**
 
-### Phar archive
+The best option for most cases is to choose  **composer package with phar only**. 
 
-To install Deployer via phar archive, run the following commands:
+Two other methods have following disadvantages:
 
-```sh
-curl -LO https://deployer.org/deployer.phar
-mv deployer.phar /usr/local/bin/dep
-chmod +x /usr/local/bin/dep
-```
+* If you will install Deployer globally as **phar archive** then it will fit your needs well until the amount of 
```

Comment:
I think it's better to omit this section at all, users don't read docs. Better just to lett what to do and it's all.